### ISSUE TITLE
Change gap between breadcrumbs

### ISSUE THREAD

**Issue by stephcathoh** [x3p0-dev/x3p0-breadcrumbs#6]
I would love an option to change the gap between breadcrumbs

**Comment by justintadlock** [comment]
There's an issue with the way `blockGap` works for blocks that don't support layout. I'll have to look up the ticket (not sure if it's fixed yet).

In the meantime, it's possible to set the gap manually via `settings.custom.x3p0Breadcrumbs.gap` in `theme.json`.

**Comment by justintadlock** [comment]
Yep, it's using a custom implementation for the gap instead of the standard WordPress `blockGap` feature. I'm guessing the author went the custom route for the same reason I opted out of adding it to this plugin.

Anyway, it's on my radar, and it'll definitely be added when `blockGap` support is fixed for custom blocks using the standard method. Here's the ticket that I'm waiting on: https://github.com/WordPress/gutenberg/issues/53155

--- 

Side note: the Breadcrumb Block plugin actually uses an outdated version of this plugin's PHP code. That reminds me that I need to contact its author and see if I can convince him to upgrade that code.
